cves/CVE-2017-14170.yml

```diff
@@ -253,10 +290,9 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: fecb3e82a4ba09dc11a51ad0961ab491881a53a1
+    note: |
+      Fixes a potential integer overflow error, which may have also been exploitable
```

Comment:
```suggestion
      Fixes a potential integer overflow error, which may have also been exploitable
  - commit: 9d00fb9d70ee8c0cc7002b89318c5be00f1bbdad
    note: |
      Fixes a potential integer underflow error, which may have also been exploitable.
      This error was fixed at the same time as the other associated fix described earlier
      in the document.
```

Reply:
I'm not sure what you're trying to point out here. This is a different commit from the fix commit, and fixes a completely separate security issue